docs/rewards/Troubleshooting.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: troubleshooting
+title: Reward Annotations Troubleshoot
```

Comment:
Let's change the title to "Rewards Troubleshooting" as users have no idea what "Annotations" mean, it's more of an internal language.